MainWindow.h

```diff
@@ -57,8 +65,11 @@ class MainWindow : public QMainWindow {
   Ui::MainWindow *ui;
 
   buffers::Project *project;
+  QPointer<RecentFiles> recentFiles;
 
   void openSubWindow(buffers::TreeNode *item);
+  void readSettings();

```

Comment:
Do you intend this to read preferences as well? Right now, it's just reading window geometry/previous run state. So this is kind of a misnomer.